src/ralph/api/serializers.py

```diff
@@ -188,6 +190,13 @@ def build_field(self, *args, **kwargs):
             field_kwargs.setdefault('style', {})['base_template'] = 'input.html'
         return field_class, field_kwargs
 
+    def save(self):
```

Comment:
please make sure that every class that is used in `save_serializer_class` is inheriting from `RalphAPISaveSerializer`